scripts/sdk.js

```diff
@@ -17,9 +17,15 @@ function writeSDK(
   commitHash,
   staticLoader,
   gppStub,
+  preprod
 ) {
   loadGPPStub(gppStub, "scripts/gpp_stub.js");
+  var _base = "https://sdk.";
   var _staging = staging ? "staging." : "";
+  if (preprod) {
+    _base = "https://sdk";
+    _staging = "-preprod."
```

Comment:
so if you have both staging and preprod toggles selected, it will always choose preprod and ignore the staging toggle?

Reply:
Yes, this is idea. Pre-prod is priority